doc/classes/NetworkedMultiplayerPeer.xml

```diff
@@ -46,7 +52,7 @@
 			<argument index="0" name="enable" type="bool">
 			</argument>
 			<description>
-				If [code]endable[/code] is true, this [code]NetworkedMultiplayerPeer[/code] will refuse new connections.
+				Returns [code]true[/code] if this [code]NetworkedMultiplayerPeer[/code] refuses new connections. Default value: [code]false[/code].
```

Comment:
Confused this with `is_refusing_new_connections`, I think?

Reply:
Ah, woops. Thanks.